pom.xml

```diff
@@ -2558,6 +2660,26 @@
       </modules>
     </profile>
 
+    <!--
+      The cloud profile enables the cloud module.
+      It does not declare the hadoop-* artifacts which
+      the cloud module pulls in; these are delegated to
+      the hadoop-x.y protocols, so permitting different
+      hadoop versions to declare different include/exclude
+      rules (especially transient dependencies).
+
+      To use this profile, the hadoop-2.7 profile must also
```

Comment:
as long as you always want to include the cloud JARS on the CP, yes. I'd kept it as a separate option to avoid forcing it in, but it would be a lot simpler if it was unified